src/toGamut.js

```diff
@@ -104,3 +112,84 @@ export default function toGamut (color, {method = defaults.gamut_mapping, space
 }
 
 toGamut.returns = "color";
+
```

Comment:
I am wondering if this would have value apart from the Color.js `toGamut` call, as an exact representation of the algorithm? As it is, it is returning in the original space, whereas the CSS algorithm returns in the destination space. We could do that conversion later, and then export this function by itself, if that would be useful.

Reply:
> We could do that conversion later, and then export this function by itself, if that would be useful.

That would be best, I think.

Reply:
I made this change in https://github.com/LeaVerou/color.js/pull/344/commits/fc6b08388b23fe99d378f96c2c9c995df86dbddf